backend/src/crd/controller/viewer/main.go

```diff
@@ -55,6 +57,8 @@ func main() {
 	if err != nil {
 		log.Fatalf("Failed to build valid config from supplied flags: %v", err)
 	}
+	cfg.QPS = float32(*clientQPS)
```

Comment:
Also a question to learn, it seems to be different between the different components how the flags are defined, `var` vs in `init` what is the reason for that?

Reply:
Same as above, how the flags defined in different components is different, I try to keep them consistent inside a component.